src/main/kubectl.ts

```diff
@@ -210,7 +210,9 @@ export class Kubectl {
       })
       file.on("close", () => {
         logger.debug("kubectl binary download closed")
-        fs.chmod(this.path, 0o755, null)
+        fs.chmod(this.path, 0o755, (err) => {
```

Comment:
Should this be done for fs.unlink() calls as well? (line 208)

Reply:
Yes, it should. Maybe we can just ignore error there since we want to reject with stream error.

Reply:
Added #618 to address these calls.